### ISSUE TITLE
appId should be a string! on Android

### ISSUE THREAD

**Issue by mcorner** [AppsFlyerSDK/appsflyer-react-native-plugin#270]
# Report

The introduction of validating parameters here:
https://github.com/AppsFlyerSDK/appsflyer-react-native-plugin/issues/249

broke android intialization.

The appId is optional for Android as stated in the docs, so for now we will try and use ''

## Plugin Version

6.2.40

## On what Platform are you having the issue?

Android

## What did you do?
```
  if (Platform.OS === 'ios') {
    options.appId = 'OUR APP ID';
  }
```

## What did you expect to happen?

initialize the sdk

## What happened instead?

throws an exception of "appId should be a string!"

**Comment by mcorner** [comment]
Also, isDebug should probably be an optional parameter with a default value of false. That broke our ios integration as well.

**Comment by artnest** [comment]
We have encountered the same issue. Our workaround is to pass an empty string for `appId` parameter for Android, however, I'm not sure if the SDK completely initializes as there appeared situations when the SDK initialization in Android goes into infinite loop and the Promise from `initSdk` on React Native side never resolves.

**Comment by amit-kremer93** [comment]
Fixed on latest release (6.2.42)
